### PR TITLE
Add Marinade's config and additional parameters

### PR DESCRIPTION
**Pull request by MjCage**

### CODE REVIEW

File: src/parser/types.ts [hunk 1]
```diff
@@ -163,6 +170,10 @@ export interface GovernorConfigRaw extends GovernanceConfigDeprecatedRaw {
    * Quarry configuration
    */
   quarry?: QuarryRaw;
+  /**
+   * Token voter configuration.
+   */
+  ["token-voter"]?: TokenVoterRaw;
```
Comment (michaelhly) [inline src/parser/types.ts:176]:
```suggestion
  token-voter?: TokenVoterRaw;
```

File: src/parser/parse.ts [hunk 2]
```diff
@@ -147,6 +161,7 @@ export const parseGovernorConfig = async (
     network: cluster,
   });
   const quarry = raw.quarry ? parseQuarry(raw.quarry) : undefined;
+  const tokenVoter = raw["token-voter"];
```
Comment (michaelhly) [inline src/parser/parse.ts:164]:
```suggestion
```

File: src/parser/types.ts (hunk 1 above)
Comment (michaelhly) [reply src/parser/types.ts:176]: ah. That makes sense. Can we come up with a different name with no `-`?

Comment (MjCage) [reply src/parser/types.ts:176]: We could also go with lock or locker, I just though that might cause confusion.

Comment (MjCage) [reply src/parser/types.ts:176]: TokenLocker would probably be the most accurate because the address belongs to the locker contract.

**Comment by michaelhly** [comment]
@MjCage Okay. I take it back. `token-locker` is fine. But can we clean up the parsing code?

File: src/parser/types.ts (hunk 1 above)
Comment (michaelhly) [reply src/parser/types.ts:176]: Actually, it's fine @MjCage. Let's just make `parse.ts` more consistent.

**Pull request merged**